quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/configuration/test/IgnoredArtifactsTest.java

```diff
@@ -74,4 +84,46 @@ public void ignoredMapStorage() {
         assertStorage.accept("file");
         assertStorage.accept("chm");
     }
+
+    @Test
+    public void jdbcH2() {
+        assertJdbc("h2", JDBC_H2);
+    }
+
+    @Test
+    public void jdbcMssql() {
+        assertJdbc("mssql", JDBC_MSSQL);
+    }
+
+    @Test
+    public void jdbcMariadb() {
+        assertJdbc("mariadb", JDBC_MARIADB);
+    }
+
+    @Test
+    public void jdbcMysql() {
+        assertJdbc("mysql", JDBC_MYSQL);
+    }
+
+    @Test
+    public void jdbcOracle() {
+        assertJdbc("oracle", JDBC_ORACLE);
+    }
+
+    @Test
+    public void jdbcPostgres() {
+        assertJdbc("postgres", JDBC_POSTGRES);
+    }
+
+    private void assertJdbc(String vendor, Set<String> notIgnored) {
```

Comment:
I think we could also check the unused DB artifacts are not present.

Reply:
Ok